manual/guides/writing/presentation.md

```diff
@@ -5,6 +5,57 @@ subtitle: "The difference between screwing around and science is writing it down
 permalink: /manual/guides/writing/presentation
 ---
 
-## Template
+## What is a Presentation
 
-A template appears on [github.](https://github.com/arfc/pres)
+A presentation, also known as an oral presentation, is typically prepared to 
+summarize or disseminate work for a number of reasons. They can serve as a 
+an accountability checkpoint for an organization funding research, or as a means
+to share discovery with a variety of technical and non-technical audiences.
+
+
+## Rules of the Presentation
+
+- After you have been invited/accepted to give an oral presentation, outline and prepare your presentation.
+- Know your audience and determine the style of presentation that suits them best
+- Remember, this presentation is an advert for your work
+- Try to spend around two minutes per slide
+- Present the body of your talk with slides titled as summaries or guiding questions
+- People will remember your figures, not your bullets (a generalization, but well founded)
+- Never put something on a slide you do not throughouly, consistently, understand
+
+
+## Writing the Presentation
+
+- All grammatical and technical errors must be eliminated per the writing checklist
+- Do not use bullets unless you are presenting an acual list
```

Comment:
This might fall under a personal preference, so we can just remove it. I do agree that there shouldn't be hanging bullets though!
```suggestion
- Do not leave a bulleted list hanging (i.e. without a title or lead-in sentence)
```